js/index.js

```diff
@@ -38,5 +38,80 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+//header
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let nav = document.querySelectorAll('a');
+nav[0].textContent = (siteContent["nav"]["nav-item-1"])
+nav[1].textContent = (siteContent["nav"]["nav-item-2"])
+nav[2].textContent = (siteContent["nav"]["nav-item-3"])
+nav[3].textContent = (siteContent["nav"]["nav-item-4"])
+nav[4].textContent = (siteContent["nav"]["nav-item-5"])
+nav[5].textContent = (siteContent["nav"]["nav-item-6"])
+
```

Comment:
Here is an example of doing it the way we talked about in stand up using the forEach method. This code loops through the nodelist you have selected and assigns values from the siteContent object.

nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});